File: sample/src/main/java/com/your_company/ui_bindings_sample/InboxActivity.java
package com.your_company.ui_bindings_sample;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.nearit.ui_bindings.NearITUIBindings;

public class InboxActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inbox);
    }

    public void openInbox(View view) {
        startActivity(NearITUIBindings.getInstance(this)
                .inboxIntentBuilder()
                .includeCoupons()
                .build()
        );
    }

    public void openInboxInActivity(View view) {
        startActivity(new Intent(this, InboxPlainActivity.class));
    }
}

File: sample-kotlin/src/main/java/com/your_company/ui_bindings_kotlin_sample/InboxActivity.kt
package com.your_company.ui_bindings_kotlin_sample

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.nearit.ui_bindings.NearITUIBindings

class InboxActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_inbox)
    }

    fun openInbox(view: View) {
        startActivity(NearITUIBindings.getInstance(this@InboxActivity)
                .inboxIntentBuilder()
                .setTitle("You got mail")
                .build())
    }

    fun openInboxInActivity(view: View) {
        startActivity(Intent(this@InboxActivity, InboxPlainActivity::class.java))
    }
}